BadPotato1007.md

```diff
@@ -0,0 +1,65 @@
+5. Use the drumbox looper and make the project opensource with instructions on how to make it on instructables
+
+
+## Budget
+
+
+| Product                   | Supplier/Link                                                                                      | Cost   |
+| ---------------           | -------------------------------------------------------------------------------------------------- | ------ |
+| 7" LCD display            | https://www.amazon.in/REES52-Display-1024X600-Function-Raspberry/dp/B07GDLB7TM/product/1957        | $81.25 |
+| Rotry Encoder             | https://robu.in/product/m274-360-degree-rotary-encoder-module-brick-sensor/                        | $0.68  |
+| Rasberry pi 3b            | https://www.olx.in/item/raspberry-pi-3b-6-pieces-brand-new-condition-iid-1702194889                | $68.75 |
+| MCP23017                  | https://robu.in/product/mcp23017-16-bit-input-output-expander-with-i2c-interface-ic-dip-28-package | $4.00  |
+| 74HC595                   | https://robu.in/product/sn74hc595n-dip16/  (owned)                                                 | $1.75  |
+| Potentiometer             | https://robu.in/product/ptv09a-4225f-b103-rotary-potentiometer/                                    | $1.30  |
+| Capacitive Sensor         | https://robu.in/product/ttp223-touch-key-module-2pcs/                                              | $6.00  |
+| 12V 2A Adapter            | Owned                                                                                              | $3.00  |
+| RGB leds                  | https://robu.in/product/ws2812b-4-x-4-rgb-led-module/                                              | $2.70  |
+| 3D printed parts          | Nearby shop                                                                                        | $10.00 |
+| Acrylic sheets            | https://cutt.ly/u2ujscX, shorturl.at/cdes5                                                         | $15.00 |
+| Speaker                   | Owned                                                                                              | $2.00  |
+| Mic                       | shorturl.at/ahJL2                                                                                  | $1.00  |
+| USB soundcard             | shorturl.at/nGZ25                                                                                  | $2.00  |
+| Midi socket               | owned                                                                                              | $7.00  |
+| Screws and other expenses |                                                                                                    | $10.00 |
+| Total                     |                                                                                                    | $217.43|
+|Funds I actually require   |                                                                                                    | $183.63|
```

Comment:
how did you arrive at this number? also, we're offering up to $250 so you're under the budget and if the PR is approved we'd be cover the full cost :)

Reply:
I already have some of the part, and can get some funds froom some place else.. Thats how I came up with those numbers..
As for the midi socket a shop near my house sells them at 550 inr, which is around $7, and other expenses covers all the glues and screws and other things I need to mke it happen :)

Reply:
ahhhh cool! @BadPotato1007 since we'll only be transferring you the amount set on your budget, could you please update so that it only has one number (the amount that you need)?

Reply:
> ahhhh cool! @BadPotato1007 since we'll only be transferring you the amount set on your budget, could you please update so that it only has one number (the amount that you need)?

Done! please check it now 
I also added shipping from olx and a $2 part that I had forgotten to add before.. please include that!